src/Discovery/Discovery.tsx

```diff
@@ -726,6 +739,11 @@ const Discovery: React.FunctionComponent<Props> = (props: Props) => {
           && (
             <div className='discovery-search-container discovery-search-container__standalone'>
               <DiscoveryMDSSearch
+                searchableTextFields={config.features.search.searchBar.searchableTextFields}
+                searchableAndSelectableTextFields={config.features.search.searchBar.searchableAndSelectableTextFields}
```

Comment:
Per comment above should this be assigned searchableAndSelectableTextFields instead of config.features.search.searchBar.searchableAndSelectableTextFields?